### PR TITLE
HW1, HW2, HW3, HW4

### PR DESCRIPTION
**Pull request by NovitskaMariia**

### CODE REVIEW

File: MariiaNovitska/HW1/Zen_of_python_count.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Zen_of_Python = """ The Zen of Python, by Tim Peters
```
Comment (kolyasalubov) [inline MariiaNovitska/HW1/Zen_of_python_count.py:1]: good job

File: MariiaNovitska/HW2/natural number.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+input_number = input("Write your number: ")
```
Comment (kolyasalubov) [inline MariiaNovitska/HW2/natural number.py:1]: good job

File: MariiaNovitska/HW2/natural number.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+a = list(input_number)
+product = int(a[0])*int(a[1])*int(a[2])*int(a[3])
+b = a[::-1]
+reverse_number = int(''.join(map(str, b)))
```
Comment (kolyasalubov) [inline MariiaNovitska/HW2/natural number.py]: look at another way
print(f'Reversed number: {a[::-1]}')

File: MariiaNovitska/HW3/variables chenge.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+a = 123
```
Comment (kolyasalubov) [inline MariiaNovitska/HW3/variables chenge.py:1]: good job

File: MariiaNovitska/HW4/temp.py [hunk 5]
```diff
@@ -0,0 +1,7 @@
+temperature = int(input("Enter the temperature in Celsius: "))
```
Comment (kolyasalubov) [inline MariiaNovitska/HW4/temp.py:1]: good job

File: MariiaNovitska/HW5_loops/task_1.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+my_list = [1, 2, 45, 999]
```
Comment (kolyasalubov) [inline MariiaNovitska/HW5_loops/task_1.py:1]: good job

File: MariiaNovitska/HW5_loops/task_1.py [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+my_list_float = []
+
+for f in  my_list:
```
Comment (kolyasalubov) [inline MariiaNovitska/HW5_loops/task_1.py:4]: for f in my_list:
    my_list_float.append(float(f))

File: MariiaNovitska/HW5_loops/task_2.py [hunk 8]
```diff
@@ -0,0 +1,17 @@
+input_number = int(input("How many Fibonacci numbers do you want to see? "))
```
Comment (kolyasalubov) [inline MariiaNovitska/HW5_loops/task_2.py:1]: good job
look at this code:


input_number = int(input("How many Fibonacci numbers do you want to see? "))

if input_number == 0:
    print("Sooo, you don't want to see Fibonacci numbers? OK, it's your choice)")
elif input_number == 1:
    print([0])
else:
    num = [0, 1]
    for _ in range(input_number - 2):
        num.append(num[-1] + num[-2])
    print(num)

File: MariiaNovitska/HW5_loops/task_3.py [hunk 9]
```diff
@@ -0,0 +1,12 @@
+number = int(input("Write your number to calculate factorial: "))
```
Comment (kolyasalubov) [inline MariiaNovitska/HW5_loops/task_3.py:1]: good job
look at this code: 


number = int(input("Write your number to calculate factorial: "))

if number < 0:
    print("It's impossible to calculate the factorial for negative numbers.")
else:
    factorial = 1
    for f in range(1, number + 1):
        factorial *= f
    print(f"{number}! = {factorial}")

File: MariiaNovitska/HW2/natural number.py (hunk 3 above)
Comment (NovitskaMariia) [reply MariiaNovitska/HW2/natural number.py]: updeted this task

File: MariiaNovitska/HW5_loops/task_1.py (hunk 7 above)
Comment (NovitskaMariia) [reply MariiaNovitska/HW5_loops/task_1.py:4]: tried this method on my PC

File: MariiaNovitska/HW5_loops/task_2.py (hunk 8 above)
Comment (NovitskaMariia) [reply MariiaNovitska/HW5_loops/task_2.py:1]: understood

File: MariiaNovitska/HW5_loops/task_3.py (hunk 9 above)
Comment (NovitskaMariia) [reply MariiaNovitska/HW5_loops/task_3.py:1]: Got it)

File: MariiaNovitska/HW6_lists/task_1.py [hunk 10]
```diff
@@ -0,0 +1,33 @@
+# з використанням циклів:
+my_list = [x for x in range (1, 11)]
+
```
Comment (kolyasalubov) [inline MariiaNovitska/HW6_lists/task_1.py:3]: good job
look at this code

even_numbers = []
odd_numbers_divisible_by_3 = []
other_numbers = []

for item in range(1, 11):
    if item % 2 == 0:
        even_numbers.append(item)
    elif item % 3 == 0:
        odd_numbers_divisible_by_3.append(item)
    else:
        other_numbers.append(item)

print(f"Even numbers: {even_numbers}")
print(f"Odd numbers divisible by 3: {odd_numbers_divisible_by_3}")
print(f"Other numbers (not divisible by 2 and 3): {other_numbers}")

File: MariiaNovitska/HW6_lists/task_2.py [hunk 11]
```diff
@@ -0,0 +1,8 @@
+user_name = input("Login: ")
+while user_name == "First":
+    print(f"Hello, {user_name}")
+    break
+else:
+    print("Wrong login")
+
+# Можливо, я не правильно зрозуміла завдання
```
Comment (kolyasalubov) [inline MariiaNovitska/HW6_lists/task_2.py:8]: a little bit

File: MariiaNovitska/HW6_lists/task_2.py [hunk 12]
```diff
@@ -0,0 +1,8 @@
+user_name = input("Login: ")
```
Comment (kolyasalubov) [inline MariiaNovitska/HW6_lists/task_2.py:1]: according to the condition of the problem, the solution should be like this

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)

**Pull request merged**